backend/grant/proposal/commands.py

```diff
@@ -76,3 +76,44 @@ def create_proposals(count):
 
     db.session.commit()
     print(f'Added {count} LIVE fake proposals')
+
+
+@click.command()
+@click.argument('dry', required=False)
+@with_appcontext
+def retire_v1_proposals(dry):
+    now = datetime.datetime.now()
+    proposals = Proposal.query.filter_by(stage="FUNDING_REQUIRED").all()
+    modified_count = 0
+
+    if not proposals:
+        print("No proposals found. Exiting...")
+        return
+
+    print(f"Found {len(proposals)} proposals to modify")
+    if dry:
+        print(f"This is a dry run. Changes will not be committed to database")
+
+    confirm = input("Continue? (y/n) ")
+
+    if confirm != "y":
+        print("Exiting...")
+        return
+
+    for p in proposals:
+        new_deadline = (now - p.date_published).total_seconds()
```

Comment:
Can all get moved into `if not dry` block below. We don't need to modify the properties on those objects in a not dry run.

Reply:
Good call, done :+1: